MapboxDirections/MBRoute.swift

```diff
@@ -68,6 +68,11 @@ open class Route: NSObject, NSSecureCoding {
             return nil
         }
         routeOptions = options
+        
+        guard let decodedRouteIdentifier = decoder.decodeObject(of: NSString.self, forKey: "routeIdentifier") as String? else {
+            return nil
```

Comment:
Oh, also, since `routeIdentifier` is optional, we should probably just swallow the lack of an identifier. No need to guard in this case.